# Robot name goes here
robot_2: # change the name
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  gazebo_ros_control:
    pid_gains:
      joint_0:
        p: 100.0
        i: 1
        d: 10.0
      joint_1:
        p: 100.0
        i: 1
        d: 10.0
  joint_0_controller:
    type: effort_controllers/JointPositionController
    joint: joint_0
    pid: {p: 100.0, i: 1, d: 10.0}
  
  joint_1_controller:
    type: effort_controllers/JointPositionController
    joint: joint_1
    pid: {p: 100.0, i: 1, d: 10.0}
  
  joint_2_controller:
    type: effort_controllers/JointPositionController
    joint: joint_2
    pid: {p: 100.0, i: 1, d: 10.0}
  
  joint_3_controller:
    type: effort_controllers/JointPositionController
    joint: joint_3
    pid: {p: 100.0, i: 1, d: 10.0}
  
  joint_4_controller:
    type: effort_controllers/JointPositionController
    joint: joint_4
    pid: {p: 100.0, i: 1, d: 10.0}
  
  joint_5_controller:
    type: effort_controllers/JointPositionController
    joint: joint_5
    pid: {p: 100.0, i: 1, d: 10.0}

 